public/scss/buttons.scss

```diff
@@ -6,12 +6,23 @@
     font-size: 1.1em;
     font-weight: bolder;
     letter-spacing: 1px;
-    padding: 0.9em 0;
+    padding: 0.9em 2em;
+    margin: 10px 0;
     width: 100%;
 
     &[disabled]:hover {
         color: $blue-light;
     }
+
+    &.center-block {
+        margin-left: 0;
+        margin-right: 0;
+    }
+
+    @media (min-width: $tablet-min) {
+        width: auto;
```

Comment:
I know we're only developing for mobile, but this is what it looks like for desktop/tablet:
![image](https://cloud.githubusercontent.com/assets/970117/25253367/799c10fa-25d5-11e7-90b2-c64e50833df4.png)